assets/js/page-enhancements.js

```diff
@@ -105,8 +105,9 @@ window.pageEnhancements = (($, document) => {
       event.preventDefault()
       const form = $(this).closest('form')
       const csrfQueryString = extractCsrfFieldAsKeyValuePairFromForm(form)
-      const formAction = `${form.attr('action')}?${csrfQueryString}`
+      const formAction = `${form.attr('action')}`
       const data = {}
+      data['_csrf'] = csrfQueryString[0].split('=')[1]
```

Comment:
It would seem the old csurf module accepted the token on GET/POST from the querystring but it should be in the form data when POSTed.

Reply:
Int tests failed without this fix?